ADAPIReposetory/implementions/Repository.cs

```diff
@@ -36,6 +36,41 @@ public void AddADObject(ADObject adObject, string adObjectType)
             }
             newObjectEntry.CommitChanges();
         }
+        } catch (Exception ex) { 
+            Console.WriteLine($"An error occurred: {ex.Message}"); 
+        }
+    }
+
+    public void ModifyADObject(ModifyModel newAdObject, string adObjectType)
+    {
+        try
+        {
+            string ldapPath = _configuration["ActiveDirectory:LDAPPath"];
+            string entryPath = $"LDAP://{newAdObject.Identifier?.Value},{ldapPath}";
```

Comment:
i want you to search the entry and modify it
you are not getting ou at this request